.travis.yml

```diff
@@ -8,4 +8,4 @@ before_install:
 script:
   # the "storage-ng-travis-ruby" script is included in the base yastdevel/storage-ng image
   # see https://github.com/yast/docker-storage-ng/blob/master/storage-ng-travis-ruby
-  - docker run -it -e TRAVIS=1 -e TRAVIS_JOB_ID="$TRAVIS_JOB_ID" yast-storage-ng-image storage-ng-travis-ruby
+  - docker run -it -e TRAVIS=1 -e TRAVIS_JOB_ID="$TRAVIS_JOB_ID" yast-storage-ng-image storage-ng-travis-ruby rpm -qa | sort
```

Comment:
I guess this `rpm` thingy is a leftover. Or do you really want to keep it there?